bundles/ch.elexis.kassenbuch/src/ch/elexis/buchhaltung/kassenbuch/BuchungsDialog.java

```diff
@@ -32,13 +36,13 @@
 import ch.rgw.tools.TimeTool;
 
 public class BuchungsDialog extends TitleAreaDialog {
-
+	private final ExecutorService executor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors());
```

Comment:
Da die recalc Methode nicht in mehrere Threads aufgeteilt wird, würde ich hier nicht einen eigenen Executor mit der Anazhl der Prozessoren anlegen. Es wird nur 1 Thread verwendet, dafür ist der default Executor ausreichend.

Diese Zeile bitte entfernen.